llarp/rpc/rpc_server.cpp

```diff
@@ -23,7 +24,15 @@ namespace llarp::rpc
 {
   RpcServer::RpcServer(LMQ_ptr lmq, AbstractRouter* r)
       : m_LMQ{std::move(lmq)}, m_Router{r}, log_subs{*m_LMQ, llarp::logRingBuffer}
-  {}
+  {
+    for (auto addr : r->GetConfig()->api.m_rpcBindAddresses)
```

Comment:
`const auto &` will prevent an extra copy construction here, a pattern i like is to only use `auto &` or `const auto &` unless a copy construction is explicitly desired. using `auto &` will let you `std::move` (steal ownership) it into use elsewhere, and `const auto &` will ensure any copy constriction you do is on purpose, not a side effect of the loop.

Reply:
const correctness is also here too as unless you explicitly want to modify things in place with the loop, it should be a `const auto &`

Reply:
fixed